linera-views/src/views/bucket_queue_view.rs

```diff
@@ -130,6 +130,7 @@ fn stored_indices<T>(stored_data: &VecDeque<(usize, Bucket<T>)>, position: usize
 /// The size `N` has to be chosen by taking into account the size of the type `T`
 /// and the basic size of a block. For example a total size of 100bytes to 10KB
 /// seems adequate.
+#[derive(Debug)]
 pub struct BucketQueueView<C, T, const N: usize> {
     context: C,
     /// The buckets of stored data. If missing, then it has not been loaded. The first index is always loaded.
```

Comment:
By reading this, we don't really understand what `(usize, Bucket<T>)` is for. Can we define a struct?